README.md

```diff
@@ -1,115 +1,143 @@
+  annotate          Annotate the build page within the Buildkite UI with text from within a Buildkite job
+  annotation        Make changes to an annotation on the currently running build
+  artifact          Upload/download artifacts from Buildkite jobs
+  env               Process environment subcommands
+  lock              Process lock subcommands
+  meta-data         Get/set data from Buildkite jobs
+  oidc              Interact with Buildkite OpenID Connect (OIDC)
+  pipeline          Make changes to the pipeline of the currently running build
+  step              Get or update an attribute of a build step
+  bootstrap         Run a Buildkite job locally
+  help              Shows a list of commands or help for one command
 
 Use "buildkite-agent <command> --help" for more information about a command.
 ```
 
 ## Dependencies
 
-The agent is fairly portable and should run out of the box on most supported platforms without extras. On Linux hosts it requires `dbus`.
+The agent is fairly portable and should run out of the box on most supported
+platforms without extras. On Linux hosts it requires `dbus`.
 
 ## Installing
 
-[The agents page](https://buildkite.com/organizations/-/agents) on Buildkite has personalised instructions, or you can refer to [the Buildkite docs](https://buildkite.com/docs/agent/v3/installation). Both cover installing the agent with Ubuntu (via apt), Debian (via apt), macOS (via homebrew), Windows and Linux.
+[The agents page](https://buildkite.com/organizations/-/agents) on Buildkite has
+personalised instructions, or you can refer to
+[the Buildkite docs](https://buildkite.com/docs/agent/v3/installation). Both
+cover installing the agent with Ubuntu (via apt), Debian (via apt), macOS (via
+homebrew), Windows and Linux.
 
 ### Docker
 
-We also support and publish [Docker Images](https://hub.docker.com/r/buildkite/agent) for the
-following operating systems. Docker images are tagged using the agent SemVer components followed
-by the operating system.
+We also support and publish
+[Docker Images](https://hub.docker.com/r/buildkite/agent) for the following
+operating systems. Docker images are tagged using the agent SemVer components
+followed by the operating system.
 
-For example, agent version 3.30.0 is published as:
+For example, agent version 3.45.6 is published as:
 
-- 3-ubuntu-20.04, tracks minor and bugfix updates in version 3 installed in Ubuntu 20.04
-- 3.30-ubuntu-20.04, tracks bugfix updates in version 3.30 installed in Ubuntu 20.04
-- 3.30.0-ubuntu-20.04, tracks the exact version installed in Ubuntu 20.04
+- 3-ubuntu-20.04, tracks minor and bugfix updates in version 3 installed in
+  Ubuntu 20.04
+- 3.45-ubuntu-20.04, tracks bugfix updates in version 3.45 installed in Ubuntu
+  20.04
+- 3.45.6-ubuntu-20.04, tracks the exact version installed in Ubuntu 20.04
 
 #### Supported operating systems
 
-- Alpine 3.12
+- Alpine 3.18
 - Ubuntu 18.04 LTS (x86_64), supported to end of life for 18.04
```

Comment:
I would like us to make a decision to change this to "supported to end of standard support for 18.04"

EoL is 10 years, standard support is 5 years.

Reply:
SGTM!